datadog-integration/modules/auth/locals.tf

```diff
@@ -1,8 +1,9 @@
 locals {
   user_group_name  = "${var.user_name}-admin"
   user_policy_name = "${var.user_name}-policy"
-  dg_name          = "dd-sch-dg"
-  dg_policy_name   = "${local.dg_name}-policy"
+  dg_sch_name      = "dd-connectorhubs"
+  dg_fn_name       = "dd-functions"
```

Comment:
same with this variable name, it might be verbose, but will nice to name these clearly